src/Giraffe/ResponseWriters.fs

```diff
@@ -157,9 +160,34 @@ type HttpContext with
         task {
             this.SetContentType "application/json; charset=utf-8"
             this.SetHttpHeader "Transfer-Encoding" "chunked"
+            let writeChunk (strOut : string) = task {
+                // Write the length in hex
+                do! this.Response.WriteAsync (sprintf "%s\r\n" (strOut.Length.ToString("X")))
+                // Write the chunk
+                do! this.Response.WriteAsync (sprintf "%s\r\n" strOut)
```

Comment:
This could probably be done all in one `sprintf`.

Reply:
I should also probably use `String.Format` instead for lower allocations and speed benefits.